package.json

```diff
@@ -42,19 +42,17 @@
     "stack-trace": "0.0.10"
   },
   "peerDependencies": {
-    "@atomist/automation-client": ">=1.1.0",
+    "@atomist/automation-client": ">=1.0.1",
```

Comment:
Why did you move that back to `1.0.1`? That won't work with the removed `apollo-codegen` and `graphql-code-generator`.

Reply:
Right, but we have to stick to 1.1.0 for the release and I just fear that we'll forget if we don't rev those peerDependencies at the same time as the devDependencies.